WordPress/Classes/ViewRelated/Me/My Profile/Gravatar/GravatarInterceptingLogosView.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+
+class GravatarInterceptingLogosView: UIView {
+
+    private enum Constants {
+        static let wordpressLogo = UIImage(named: "wordpress-circular")
+        static let jetpackLogo = UIImage(named: "jetpack-circular")
+        static let gravatarLogo = UIImage(named: "gravatar-circular")
+        static let intersectionAmount: CGFloat = 12.0
+        static let logoSize: CGSize = .init(width: 30, height: 30)
+    }
+
+    private lazy var appLogoImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.image = appLogo
+        imageView.contentMode = .scaleAspectFit
+        imageView.widthAnchor.constraint(equalToConstant: Constants.logoSize.width).isActive = true
+        imageView.heightAnchor.constraint(equalToConstant: Constants.logoSize.height).isActive = true
+        return imageView
+    }()
+
+    private lazy var gravatarLogoImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.image = Constants.gravatarLogo
+        imageView.contentMode = .scaleAspectFit
+        imageView.widthAnchor.constraint(equalToConstant: Constants.logoSize.width).isActive = true
+        imageView.heightAnchor.constraint(equalToConstant: Constants.logoSize.height).isActive = true
+        return imageView
+    }()
+
+    private var appLogo: UIImage? {
+        return AppConfiguration.isJetpack ? Constants.jetpackLogo : Constants.wordpressLogo
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        addSubview(appLogoImageView)
+        addSubview(gravatarLogoImageView)
+        gravatarLogoImageView.layer.zPosition = 1
```

Comment:
Why was this necessary? Would `appLogoImageView` have a zPosition 0 and gravatarLogoImageView 1 by default due to the order we add them to the view hierarchy?

Reply:
According to designs the Gravatar logo should appear on top, but I think we don't need to set the zPosition for that. Let me revisit this.

Reply:
Replaced with bringSubviewToFront.